Cpanel/Tests/Cpanel/Core/AbstractTest.php

```diff
@@ -1189,15 +1194,20 @@ public function testProtectedArrayTypeThrowsOnBadInput($input, $expectExcep, $ex
      */
     public function testProtectedArrayTypeReturnRightType($input, $expectExcep, $expectType = null)
     {
-        if (!is_null($expectType)) {
-            $cs = $this->getCS();
-            $rprop = new ReflectionMethod($cs, 'arrayType');
-            $rprop->setAccessible(true);
+        $cs = $this->getCS();
+        $rprop = new ReflectionMethod($cs, 'arrayType');
+        $rprop->setAccessible(true);
+        if ($expectExcep) {
+            $this->expectException('Exception');
+        }else{
             $this->assertEquals($expectType, $rprop->invoke($cs, $input));
         }
+        $rprop->invoke($cs, $input);
+
     }
     public function testLegacySetUnderscoreTypeUsesStoreAdapter()
     {
+        $rObj = $this->getRObj();
```

Comment:
I don't see `$rObj` being used so what is the purpose of lines like this?

Reply:
hmm. seems to have been a copy-pasta issue will resolve.